app/services/partnerships/create.rb

```diff
@@ -100,5 +102,9 @@ def has_previous_cohort_and_matching_lead_provider?
 
       previous_year_lead_provider != lead_provider && school.school_cohorts.find_by(cohort: cohort_record).blank?
     end
+
+    def duplicate_delivery_partner_request?
```

Comment:
thanks for adding this! only comment is we will get duplicate errors with `recruited_by_other_provider` if there is another existing partnership, maybe we can only return this if the other one fails?

Reply:
Sure, added a check to see if we already have an error.